--- build/moz.configure/rust.configure.orig	2018-10-16 13:12:38.579641420 +0200
+++ build/moz.configure/rust.configure	2018-10-16 13:11:55.771598359 +0200
@@ -205,13 +205,22 @@ def rust_triple_alias(host_or_target):
             os.write(in_fd, source)
             os.close(in_fd)
 
-            cmd = [
-                rustc,
-                '--crate-type', 'staticlib',
-                target_arg,
-                '-o', out_path,
-                in_path,
-            ]
+            if 'XBPS_CROSS_BASE' in os.environ and os.uname()[4] != str(host_or_target.cpu):
+                cmd = [
+                    rustc,
+                    '--crate-type', 'staticlib',
+                    target_arg,
+                    '-o', out_path,
+                    in_path,
+                    "--sysroot="+os.environ['XBPS_CROSS_BASE']+"/usr"
+                ]
+            else:
+                cmd = [
+                    rustc,
+                    '--crate-type', 'staticlib',
+                    '-o', out_path,
+                    in_path,
+                ]
 
             def failed():
                 die(dedent('''\
