sd_fusing tweaks

--- sd_fuse/sd_fusing.sh.orig	2018-11-17 11:16:15.040562993 +0100
+++ sd_fuse/sd_fusing.sh	2018-11-17 11:19:15.993050356 +0100
@@ -24,11 +24,11 @@ if [ ! -f $UBOOT ]; then
         exit 1
 fi
 
-sudo dd if=$BL1 of=$1 conv=fsync,notrunc bs=1 count=442
-sudo dd if=$BL1 of=$1 conv=fsync,notrunc bs=512 skip=1 seek=1
-sudo dd if=$UBOOT of=$1 conv=fsync,notrunc bs=512 seek=97
+dd if=$BL1 of=$1 conv=fsync,notrunc bs=1 count=442
+dd if=$BL1 of=$1 conv=fsync,notrunc bs=512 skip=1 seek=1
+dd if=$UBOOT of=$1 conv=fsync,notrunc bs=512 seek=97
 
 sync
 
-sudo eject $1
-echo Finished.
+eject $1
+echo "Successfully wrote U-Boot to $1"
